tests/bluetooth/tester/src/btp/btp_bas.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2024 Demant A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef BTP_BAS_H_
+#define BTP_BAS_H_
+
+#include <stdint.h>
+#include <stddef.h>
+
+/* BTP BAS Service Opcodes */
+#define BTP_BAS_SET_BATTERY_LEVEL             0x02
+#define BTP_BAS_BLS_SET_BATTERY_PRESENT       0x03
```

Comment:
Why do they start from `0x02`?

Reply:
Because in AutoPTS, 0x01 is reserved for XXX_READ_SUPPORTED_COMMANDS.

Reply:
Aha. Shouldn't we define and implement a `BTP_BASS_READ_SUPPORTED_COMMANDS` then?

Reply:
@sjanc What's your input here? :)

Reply:
it should be implemented,  although currently autopts is not making use of it we have a long term goal to start using that to provide more generic WID handlers based on what is supported by IUT 

and it is just a few lines of code, you can just copy paste implementation from other services and update bitmask for BAS commands